mailchimp/mailchimp-authenticator/org.wso2.carbon.identity.authenticator/pom.xml

```diff
@@ -0,0 +1,96 @@
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.carbon.identity.authenticator</groupId>
+    <artifactId>org.wso2.carbon.identity.authenticator.mailchimp</artifactId>
+    <version>1.0.0</version>
+
+    <packaging>bundle</packaging>
+    <name>WSO2 Carbon - Authenticator Library For MailChimp</name>
+    <url>http://wso2.org</url>
+    <parent>
+        <groupId>org.wso2.carbon</groupId>
+        <artifactId>is-connectors</artifactId>
+        <version>1.0.0</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+    <dependencies/>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.felix</groupId>
+                    <artifactId>maven-bundle-plugin</artifactId>
+                    <version>2.3.5</version>
+                    <extensions>true</extensions>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.0</version>
+                <configuration>
+                    <source>1.7</source>
+                    <target>1.7</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>org.wso2.carbon.identity.authenticator.mailChimp.internal</Private-Package>
+                        <Import-Package>
+                            org.apache.commons.logging.*; version="1.0.4",
+                            org.osgi.framework,
+                            org.wso2.carbon.identity.application.authentication.framework.*,
+                            javax.servlet,
+                            javax.servlet.http,
+                            org.apache.oltu.oauth2.*; version="${oltu.package.import.version.range}",
+                            *;resolution:=optional
+                        </Import-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.authenticator.mailChimp.internal,
+                            org.wso2.carbon.identity.authenticator.*
```

Comment:
Export packages should have version specified as a best practise. Please fix these type of issues throughout in all your PR's.